.appveyor.yml

```diff
@@ -21,6 +21,9 @@ environment:
       - {PYVER: "35", PYTHON_ARCH: "64"}
       - {PYVER: "36", PYTHON_ARCH: "32"}
       - {PYVER: "36", PYTHON_ARCH: "64"}
+      - {PYVER: "37", PYTHON_ARCH: "32"}
+      - {PYVER: "37", PYTHON_ARCH: "64"}
+
```

Comment:
There is a failure on appveyor: I think it requires at least

    - IF "%PYVER%"=="37" SET VS_VER=14.0

in the `init` stanza, and hope for the best. Can you try that please?